ligo/src/proposal.mligo

```diff
@@ -80,7 +80,7 @@ let unfreeze (tokens, addr, ledger, total_supply : nat * address * ledger * tota
   let (ledger, total_supply) = credit_to (tokens, addr, unfrozen_token_id, ledger, total_supply) in
   (ledger, total_supply)
 
-let add_proposal (propose_params, store : propose_params * storage): storage =
+let add_proposal (propose_params, fixed_fee_in_token, store : propose_params * nat * storage): storage =
```

Comment:
`fixed_fee_in_token` seems unnecessary, as it can be extracted from `store` itself, am I right?

Reply:
Nice catch, thx! Initially I put it to `config` instead of store, and then I remembered it had to be modifiable, so I made it a part of store but forgot to remove the extra parameter.